automl/dataset_get.go

```diff
@@ -15,7 +15,12 @@
 // Package automl contains samples for Google Cloud AutoML API v1beta1.
 package automl
 
-// [START automl_get_dataset]
+// [START automl_language_entity_extraction_get_dataset]
```

Comment:
I mean technically all of the automl samples should all have been on the v1 library not the v1beta1

Reply:
That was my mistake, fixing that in #1122

Reply:
Sorry, I meant _all_ of the imports. Not just the API import. The reason the standard is to use separate files (https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#one-file-per-sample) is so that we're 100% sure the sample will work when copied and pasted from cloud.google.com.

Reply:
It's also very easy to mess up start/end tags when going in and out of functions (looking at the rest of the PR). I'm not sure every region is correct, so I'd prefer separate files, if you don't mind.

Reply:
If the go folks want to maintain a bunch of additional files, that is fine. The goal of this was to move us away from maintaining 1400 files to about 400 for AutoML snippets.

Reply:
Y'all will be maintaining the samples. So, if you prefer to do it this way, it seems correct from what I can tell. Let's go ahead with this as-is and we can change it later if needed. Sorry for the back and forth.